dietpi/dietpi-software

```diff
@@ -953,10 +953,10 @@ _EOF_
 		#------------------
 		software_id=183
 
-		aSOFTWARE_NAME[$software_id]='Bitwarden_RS'
+		aSOFTWARE_NAME[$software_id]='Vaultwarden'
```

Comment:
Shall we use capital V here? In the repository lower-case v is used everywhere: https://github.com/dani-garcia/vaultwarden
... checking forum: https://vaultwarden.discourse.group/

Jep, let's follow the way the developer spells it :).

Reply:
I was not 100% sure on it. Had same question in mind. 🤣

Reply:
ok should be done now